test/server/cards/01_SOR/units/LeiaOrganaDefiantPrincess.spec.js

```diff
@@ -0,0 +1,50 @@
+describe('Leia Organa, Defiant Princess', function() {
+    integration(function() {
+        describe('Leia Organa, Defiant Princess\'s when played ability', function() {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['leia-organa#defiant-princess'],
+                        groundArena: [{ card: 'atst', exhausted: true }],
+                        spaceArena: [{ card: 'cartel-spacer', exhausted: false }],
```

Comment:
Don't need to explicitly mark `exhausted: false` on any of the units, it's the default

Reply:
Following up on this - is it intentional to leave the explicit `exhausted: false` here? Since tests serve as documentation for writing other tests I would prefer to omit properties that aren't needed so that other people don't see them and think that they are needed.

If you prefer to keep them here to be explicit that's fine, in that case though can we get a comment explaining that's the reason and that the parameter is already false by default

Reply:
I changed it so that only the exhausted units explicitly have the exhausted property.